src/Propulsion.SqlStreamStore/SqlStreamStoreSource.fs

```diff
@@ -0,0 +1,57 @@
+namespace Propulsion.SqlStreamStore
+
+open System
+open SqlStreamStore
+
+open StreamReader
+open LockedIngester
+
+type SqlStreamStoreSource =
+
+    static member Run
+            (logger : Serilog.ILogger,
+             store: IStreamStore,
+             ledger: ILedger,
```

Comment:
Ledger implementation is pluggable as discussed before. Ledger.fs file has two implementations of it, SQL one and in-memory one.